hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAirdropHandler.java

```diff
@@ -255,17 +269,32 @@ final var record =
         }
     }
 
+    /**
+     * When we do an airdrop we need to check if there are custom fees that needs to be paid by the receiver.
+     * If there are, an error is returned.
+     * However, there is an exception to this rule - if the receiver is the fee collector or the treasury account
+     * they are exempt from paying the custom fees thus we don't need to check if there are custom fees.
+     * This method returns if the receiver is the fee collector or the treasury account.
+     */
+    private static boolean skipCustomFeeValidation(Token token, AccountID receiverId) {
+        for (var customFee : token.customFees()) {
+            if (CustomFeeExemptions.isPayerExempt(customFeeMetaFrom(token), customFee, receiverId)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
Nit, but I think it would be easier to read as one-liner in this case.
```suggestion
private static boolean skipCustomFeeValidation(@NonNull final Token token, @NonNull final AccountID receiverId) {
    return token.customFees().stream()
        .anyMatch(customFee -> CustomFeeExemptions.isPayerExempt(customFeeMetaFrom(token), customFee, receiverId));
}
```

Reply:
Much better! Thanks